azure-mgmt-storage/azure/mgmt/storage/storage_management_client.py

```diff
@@ -241,3 +257,16 @@ def usage(self):
         else:
             raise NotImplementedError("APIVersion {} is not available".format(api_version))
         return OperationClass(self._client, self.config, Serializer(self._models_dict(api_version)), Deserializer(self._models_dict(api_version)))
+
+    @property
+    def usages(self):
```

Comment:
Why does this new api version have `usages` instead of `usage`?
Is this a spelling mistake or is this a conscious change?